examples/nextjs/pages/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Close } from '@datapunt/asc-assets'
+import GlobalStyle from '@datapunt/asc-ui/lib/components/GlobalStyle'
+import Header from '@datapunt/asc-ui/lib/components/Header'
+import Button from '@datapunt/asc-ui/lib/components/Button'
+import Typography from '@datapunt/asc-ui/lib/components/Typography'
+
+export default function Home() {
+  return (
+    <>
+      <GlobalStyle />
```

Comment:
Should we not be including `GlobalStyle` in the `App` component?

Reply:
will do!